rholang/src/test/scala/coop/rchain/rholang/interpreter/CostAccountingReducerTest.scala

```diff
@@ -66,28 +69,35 @@ class CostAccountingReducerTest extends FlatSpec with Matchers with TripleEquals
   }
 
   it should "stop if OutOfPhloError is returned from RSpace" in {
-    val tuplespaceAlg = new Tuplespace[Task] {
+
+    val iSpace = new ISpaceStub[
+      Task,
+      Par,
+      BindPattern,
+      ListParWithRandom,
+      ListParWithRandom,
+      TaggedContinuation
+    ] {
       override def produce(
```

Comment:
are you sure you only wanted to override the produce method? Previously consume was stubbed as well... I guess, if the test passes - it's OK

Reply:
Consume was previously unused. Only a send is evaluated, so only a produce is needed.